pyrate/core/phase_closure/plot_closure.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
```

Comment:
I needed to load/install matplotlib in order to run the code. I guess one advantage of PyRate so far has been that matplotlib is not required for the core parts. While I think that it is good to have it now for testing, we may want to try and avoid matplotlib when merging this branch into master and do the plotting differently or in a separate 'plotting tool' such as it is done for Chandra's time series plotting functionality (see python3 utils/plot_time_series.py).

Reply:
I agree with @tfuhrmann. The plot_closure.py should be changed to a standalone script in the `utils` directory that a user can choose to run (or not) as a post-processing step. This keeps `matplotlib` dependency out of the mainline code (currently `matplotlib` is only included in the `requirements-plot.txt` dependency file).

Reply:
@basaks implemented in 67c02a1 that closure loop plotting is done *if* matplotlib is in the current env, otherwise it is skipped without breaking. This is a reasonable solution to the above